x-pack/test/saved_object_tagging/functional/tests/maps_integration.ts

```diff
@@ -127,7 +131,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       it('allows to select tags for an existing map', async () => {
         await listingTable.clickItemLink('map', 'map 4 (tag-1)');
 
-        await PageObjects.maps.saveMap('map 4 (tag-1)', false, ['tag-3']);
+        await PageObjects.maps.saveMap('map 4 (tag-1)', true, ['tag-3']);
```

Comment:
Why did this boolean value change?

Reply:
This signature of `PageObjects.maps.saveMap` changed, so instead of the parameter being `uncheckReturnToOriginModeSwitch` it was changed to be the opposite `redirectToOrigin`. This required all uses of `PageObjects.maps.saveMap` to switch the parameter to reflect the new use of the parameter. `

redirectToOrigin` is what is used in PageObjects.visualize.setSaveModalValues so it made sense to update our signatures to be consistent.